torch/testing/_internal/common_methods_invocations.py

```diff
@@ -4547,6 +4547,38 @@ def _tensor(shape, dtype=dtype, low=None, high=None):
 
     return [SampleInput(tensor, args=args) for tensor, args in test_cases]
 
+def sample_inputs_grid_sample(op_info, device, dtype, requires_grad, **kwargs):
+    def make_tensor_(shape):
+        return make_tensor(shape, device=device, dtype=dtype, requires_grad=requires_grad)
+
+    batch_size = 2
+    num_channels = 3
+    modes = ("bilinear", "nearest")
+    align_cornerss = (False, True)
```

Comment:
"cornerss" -> "corners

Reply:
How are we going to differentiate it from the loop variable? Basically, we need the plural of `align_corners` here. I'm fully aware that adding an extra `s` is not grammatically correct, but it seems like an easy way out here.

Reply:
Oh OK. That's fine, my mistake